node/parachain/src/cli.rs

```diff
@@ -98,7 +98,7 @@ pub struct RunCmd {
 
 	/// Public identity for participating in staking and receiving rewards
 	#[structopt(long, parse(try_from_str = parse_h160))]
-	pub account_id: Option<H160>,
+	pub author_id: Option<H160>,
```

Comment:
this is a good name change, need to recompile the node for the tests to pass with it

Reply:
never mind, tests are failing because changes in parachain need to be made in standalone as well, I'll do it